driver/controller.go

```diff
@@ -72,7 +72,9 @@ func (cs *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 		// Volume Availability - Default is nova
 		volAvailability = req.GetParameters()["availability"]
 	}
-
+	volPVCName := req.GetParameters()["csi.storage.k8s.io/pvc/name"]
```

Comment:
This may cause `Invalid memory address or nil pointer dereference` if csi-provisioner did not enable option `--extra-create-metadata`